Cargo.toml

```diff
@@ -271,7 +271,7 @@ postgres-openssl = { version = "0.5.0", default-features = false, features = ["r
 pulsar = { version = "4.1", default-features = false, features = ["tokio-runtime"], optional = true }
 rand = { version = "0.8.4", default-features = false, features = ["small_rng"] }
 rand_distr = { version = "0.4.2", default-features = false }
-rdkafka = { version = "0.27.0", default-features = false, features = ["tokio", "libz", "ssl", "zstd"], optional = true }
+rdkafka = { git = "https://github.com/fede1024/rust-rdkafka", rev = "7a2355f03c9aee8d4544e89bc4b0b0b6b96826e0", default-features = false, features = ["tokio", "libz", "ssl", "zstd"], optional = true }
```

Comment:
This is a commit from upstream that fixes a segfault, if I remember correctly, but hasn't yet been released?

Reply:
Correct